indigo_content_api/tests/v2/test_content_api.py

```diff
@@ -221,10 +221,27 @@ def test_published_toc_sections_with_headings(self):
         self.assertEqual(response.status_code, 200)
         self.assertEqual(response.accepted_media_type, 'application/json')
         self.assertEqual(response.data['toc'], [
-            {'id': 'sec_1', 'type': 'section', 'num': '1.',
-             'component': 'main', 'subcomponent': 'section/1',
-             'url': 'http://' + self.api_host + self.api_path + '/akn/za/act/2010/1/eng/!main/section/1',
-             'heading': 'Foo', 'title': '1. Foo'}])
+            {
```

Comment:
This implies that the TOC for all existing works will now always go below the section level. We need to work out what the impact of this for our existing websites.